akka-actor-typed/src/main/scala/akka/actor/typed/internal/Restarter.scala

```diff
@@ -124,6 +113,16 @@ import akka.actor.typed.scaladsl.Actor
 @InternalApi private[akka] final class Resumer[T, Thr <: Throwable: ClassTag](
   override val behavior: Behavior[T], override val loggingEnabled: Boolean) extends Supervisor[T, Thr] {
 
+  def init(ctx: ActorContext[T]) =
+    try {
+      wrap(Behavior.validateAsInitial(Behavior.undefer(behavior, ctx)), afterException = false)
+    } catch {
+      case NonFatal(ex: Thr) ⇒
+        // exception starting the actor, log and try again
+        log(ctx, ex)
+        wrap(behavior, afterException = true).init(ctx)
```

Comment:
what does this mean? will it retry forever?

Reply:
Yes, probably a bad idea here as well. A bit mitigated with wrapping with `ActorInitializationError` as that would at least have a explicit separate type, but `resume[Exception]` would still be dangerous.